src/core/lib/security/credentials/s2a/s2a_credentials.cc

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "src/core/lib/security/credentials/s2a/s2a_credentials.h"
+#include <grpc/support/port_platform.h>
+#include "src/core/lib/security/security_connector/s2a/s2a_security_connector.h"
+
+namespace experimental {
+
+#define GRPC_S2A_CREDENTIALS_TYPE "s2a"
+
+grpc_s2a_credentials::grpc_s2a_credentials(
+    const grpc_s2a_credentials_options* options,
+    const char* handshaker_service_url)
```

Comment:
Can we place `handshaker_service_url` inside `grpc_s2a_credentials_options`?

Reply:
Good call - added it as a member variable.